arduino/sketch_read_transmit_from_sd/sketch_read_transmit_from_sd.ino

```diff
@@ -47,6 +46,7 @@ void readFromSDCardToSerialOutputLineByLine(File file) {   // TODO: delay value
       // TODO: Delay can be added here to transmit data over the serial channel for a specified constant rate.
```

Comment:
Add delay here. `Delay(10)` (Just like the todo comment says)

Reply:
:thinking: The `delay()` seems to be creating some problem in reading the data and processing. Also it is taking a hell lot of more time to finish the recording. So as without the delay we didn't had a problem, I have removed it.

Reply:
Hell lot of time when delay(1000) or delay(10)?

Reply:
When delay(10). I think because we kept it within the loop unlike delay(1000) which was outside and got executed only once. Anyways, I have removed the `delays` here as I have checked the recorded dataset (screenshot attached) and there isn't any problem without them.